cli/internal/packagemanager/packagemanager.go

```diff
@@ -149,3 +152,11 @@ func (pm PackageManager) GetWorkspaces(rootpath fs.AbsolutePath) ([]string, erro
 func (pm PackageManager) GetWorkspaceIgnores(rootpath fs.AbsolutePath) ([]string, error) {
 	return pm.getWorkspaceIgnores(pm, rootpath)
 }
+
+// CanPrune returns if turbo can produce a pruned workspace. Can error if fs issues occur
+func (pm PackageManager) CanPrune(projectDirectory fs.AbsolutePath) (bool, error) {
+	if canPrune := pm.canPrune; canPrune != nil {
```

Comment:
I don't think you need `canPrune` here. Is this equivalent to `if pm.canPrune != nil {` ?